buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -165,6 +209,7 @@
       "alwaysRun": false,
       "displayName": "Update versions repository",
       "timeoutInMinutes": 0,
+      "condition": "and(succeeded(), eq(variables['PublishToMyGet'], 'True'), eq(variables['PB_ConfigurationGroup'], 'Release'))",
```

Comment:
I don't think conditioning this on PublishToMyGet is correct, as we will likely want to publish to versions repo if we publish to any public feed. However given we are only restoring for myget right now I'm OK with it.